spec/jobs/create_feedback_session_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe CreateFeedbackSessionJob, type: :job do
+  describe '#perform' do
+    let(:provider_user) { create(:user) }
+    let(:receiver_user) { create(:user) }
+    let(:feedback_session_date) { Time.now.strftime("%Y-%m-%d") }
+    let(:tag_names) { ['Tag_1', 'Tag_2'] }
+
+    it "creates a feedback session and associates tags" do
+      expect {
+        CreateFeedbackSessionJob.new.perform(
+          provider_user,
+          receiver_user,
+          feedback_session_date,
+          tag_names
+        )
+      }.to change(FeedbackSession, :count).by(1)
+        .and change(SessionTag, :count).by(tag_names.length)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/MultilineMethodCallIndentation: Align .and with .by on line 18.